cdap-docs/admin-manual/source/operations/preferences.rst

```diff
@@ -119,6 +119,14 @@ the argument will be set for all programs that use ``myTable``. In order to set
 program ``myMR``, the dataset argument can be nested inside the program scope as
 ``mapreduce.myMR.dataset.myTable.cache.seconds=30``.
 
+Overriding Logical Start Time with Runtime Arguments
+====================================================
+
+The runtime argument ``logicalStartTime`` can be used to override the logical start time of a Workflow, MapReduce,
+or Spark program that has been manually started.  Instead of using the current time as the logical start time,
+programs started this way will instead use the value given in the runtime argument.  The value is expected to be
```

Comment:
What about:
```
Instead of using the current time as the logical start time,
programs started manually with this argument will instead use the value given in the runtime argument.